twitter/models.py

```diff
@@ -186,6 +192,33 @@ def update_or_create_tweet(self, tweet_json):
         }
         return results
 
+
+    def retrieve_tweets_cached_locally(self):
+        tweet_on_stage = Tweet()
+        tweet_found = False
+        success = False
+        status = "TWEET_NOT_FOUND"
+
+        tweet_list = []
+
+        try:
+            tweet_list_query = Tweet.objects.all()
+            tweet_list = list(tweet_list_query)
```

Comment:
I would recommend storing "organization_we_vote_id" with each Tweet, and then you can add a filter & sort like this:
tweet_list_query = tweet_list_query.filter(organization_we_vote_id__iexact=organization_we_vote_id)
tweet_list_query = tweet_list_query.order_by('date_published').reverse()